src/devices/Mcp3xxx/samples/README .md

```diff
@@ -0,0 +1,116 @@
+        Thread.Sleep(500);
+    }
+}
+```
+
+The following pin layout can be used (also shown in a [fritzing diagram](rpi-trimpot-spi.fzz)):
+
+* MCP3008 VDD to RPi 3.3V
+* MCP3008 VREF to RPi 3.3V
+* MCP3008 AGND to RPi GND
+* MCP3008 DGND to RPi GND
+* MCP3008 CLK to RPi SCLK
+* MCP3008 DOUT to RPi MISO
+* MCP3008 DIN to RPi MOSI
+* MCP3008 CS/SHDN to RPi CE0
+
+![Raspberry Pi Breadboard diagram](rpi-trimpot_spi.png)
+
+
+## Accessing the MCP3008 via GPIO
+
+You can also access the MCP3008 via GPIO pins, implementing SPI manually. This method is referred to as [bit-banging](https://en.wikipedia.org/wiki/Serial_Peripheral_Interface#Example_of_bit-banging_the_master_protocol).
+
+You can use the following code to [access the MCP3008 via GPIO](Mcp3008.Sample.cs):
+
+```csharp
+using (SpiDevice spi = new SoftwareSpi(clk: 18, miso: 23, mosi: 24, cs: 25))
+using (Mcp3008 mcp = new Mcp3008(spi))
+{
+    while (true)
+    {
+        double value = mcp.Read(0);
+        value = value / 10.24;
+        value = Math.Round(value);
+        Console.WriteLine($"{value}%");
+        Thread.Sleep(500);
+    }
+}
+```
+
+The following pin layout can be used (also shown in a [fritzing diagram](rpi-trimpot-gpio.fzz)):
+
+* MCP3008 VDD to RPi 3.3V
+* MCP3008 VREF to RPi 3.3V
+* MCP3008 AGND to RPi GND
+* MCP3008 DGND to RPi GND
+* MCP3008 CLK to RPi pin 18
+* MCP3008 DOUT to RPi pin 23
+* MCP3008 DIN to RPi pin 24
+* MCP3008 CS/SHDN to RPi pin 25
+
+![Raspberry Pi Breadboard diagram](rpi-trimpot_gpio.png)
+
+## Processing the data
+
+Independent of the way in which you access the MCP3008 chip, the code to process its results is the same, which follows.
+
+```csharp
+while (true)
+{
+    double value = mcp.Read(0);
+    value = value / 10.24;
+    value = Math.Round(value);
+    Console.WriteLine(value);
+    Thread.Sleep(500);
+}
+```
+
+The chip is 10-bit, which means that it will generate values from 0-1023 (recall that 2^10 is 1024). We can transform the value to a more familiar 0-100 scale by dividing the 10-bit value by 10.24.
+
+## Hardware elements
+
+The following elements are used in this sample:
+
+* [Potentiometer](https://www.adafruit.com/product/356)
+* [MCP3008](https://www.adafruit.com/product/856)
+
+## References
+
+The sample is based on following resources:
+
+* [Analog Inputs for Raspberry Pi Using the MCP3008](https://learn.adafruit.com/reading-a-analog-in-and-controlling-audio-volume-with-the-raspberry-pi) 
+* [Raspberry Pi Analog to Digital Converters](https://learn.adafruit.com/raspberry-pi-analog-to-digital-converters).
+* [Raspbery Pi Analog Input with MCP3008](https://gist.github.com/ladyada/3151375)
+* [MCP3008.py](https://github.com/adafruit/Adafruit_Python_MCP3008/blob/master/Adafruit_MCP3008/MCP3008.py)
+
+Major thanks to [Adafruit](https://adafruit.com) for providing python implementations, which were ported to C# for this sample.
+
+See [Using .NET Core for IoT Scenarios](../README.md) for more samples.
```

Comment:
please remove Mcp3008 folder since it's now included in here

Reply:
@krwq
I believed I have have addressed all the comments and I have posted another PR update,